src/controllers/krate/delete.rs

```diff
@@ -116,6 +118,23 @@ pub async fn delete_crate(path: CratePath, parts: Parts, app: AppState) -> AppRe
     })
     .await?;
 
+    let email_future = async {
+        if let Some(recipient) = user.email(&mut conn).await? {
+            let email = CrateDeletionEmail {
+                user: &user.gh_login,
+                krate: &crate_name,
+            };
+
+            app.emails.send(&recipient, email).await?
+        }
+
+        Ok::<_, anyhow::Error>(())
+    };
```

Comment:
Curiosity, rather than any actual change required: was there a reason for doing this as an `async` block instead of an `async fn`?

Reply:
it would have been a fn with quite a few parameters and implementing it as a block seemed easier 😅